lib/main.dart

```diff
@@ -32,6 +32,7 @@ class MyApp extends StatelessWidget {
           title: const Text(title),
         ),
         body: ListView.builder(
+          shrinkWrap: true,
           itemCount: items.length,
           itemBuilder: (context, index) {
             final item = items[index];
```

Comment:
MyApp should reside in it's own class file

Reply:
Okay I will create new class for it